app/src/main/kotlin/com/google/samples/apps/nowinandroid/ui/interests2pane/InterestsListDetailScreen.kt

```diff
@@ -132,3 +153,59 @@ private fun <T> ThreePaneScaffoldNavigator<T>.isListPaneVisible(): Boolean =
 @OptIn(ExperimentalMaterial3AdaptiveApi::class)
 private fun <T> ThreePaneScaffoldNavigator<T>.isDetailPaneVisible(): Boolean =
     scaffoldValue[ListDetailPaneScaffoldRole.Detail] == PaneAdaptedValue.Expanded
+
+/**
+ * This is a direct clone of [calculateStandardPaneScaffoldDirective] with the only change of
+ * passing 0 content padding to the panes.
+ */
+@OptIn(ExperimentalMaterial3AdaptiveApi::class)
+private fun calculateNoContentPaddingScaffoldDirective(
+    windowAdaptiveInfo: WindowAdaptiveInfo,
+    verticalHingePolicy: HingePolicy = HingePolicy.AvoidSeparating,
+): PaneScaffoldDirective {
+    val maxHorizontalPartitions: Int
+    val verticalSpacerSize: Dp
+    when (windowAdaptiveInfo.windowSizeClass.windowWidthSizeClass) {
+        WindowWidthSizeClass.COMPACT -> {
+            maxHorizontalPartitions = 1
+            verticalSpacerSize = 0.dp
+        }
+        WindowWidthSizeClass.MEDIUM -> {
+            maxHorizontalPartitions = 1
+            verticalSpacerSize = 0.dp
+        }
+        else -> {
+            maxHorizontalPartitions = 2
+            verticalSpacerSize = 24.dp
+        }
+    }
+    val maxVerticalPartitions: Int
+    val horizontalSpacerSize: Dp
+
+    if (windowAdaptiveInfo.windowPosture.isTabletop) {
+        maxVerticalPartitions = 2
+        horizontalSpacerSize = 24.dp
+    } else {
+        maxVerticalPartitions = 1
+        horizontalSpacerSize = 0.dp
+    }
+
+    return PaneScaffoldDirective(
+        PaddingValues(0.dp),
```

Comment:
I think this was removed in alpha10, should we bump the version?

Reply:
Updated, with the latest changes this also simplified the implementation greatly.